### PR TITLE
[explorer] - update epochs table to show inflow

### PR DESCRIPTION
**Pull request by mamos-mysten**
## Description 

Update Epochs Table to show Storage Fund Net Inflow in place of Storage Revenue

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Comment by mamos-mysten** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11195** <a href="https://app.graphite.dev/github/pr/MystenLabs/sui/11195" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/MystenLabs/sui/11195?utm_source=stack-comment).

File: apps/explorer/src/utils/getStorageFundFlow.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type EndOfEpochInfo } from '@mysten/sui.js';
+
+export function getEpochStorageFundFlow(endOfEpochInfo: EndOfEpochInfo | null) {
```
Comment (Jordan-Mysten) [inline apps/explorer/src/utils/getStorageFundFlow.ts:6]: Thanks for generalizing this!

File: apps/explorer/src/utils/getStorageFundFlow.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          BigInt(endOfEpochInfo.leftoverStorageFundInflow)
+        : null;
+
+    const fundOutflow = endOfEpochInfo
+        ? BigInt(endOfEpochInfo.storageRebate)
+        : null;
+
+    const netInflow =
+        fundInflow && fundOutflow ? fundInflow - fundOutflow : null;
+    console.log(netInflow);
```
Comment (Jordan-Mysten) [inline apps/explorer/src/utils/getStorageFundFlow.ts:19]: Remove?

**Review by Jordan-Mysten — APPROVED** [review]
One nit about console log but LGTM

**Pull request merged**